Source/Core/DolphinWX/Debugger/RegisterView.cpp

```diff
@@ -513,6 +513,7 @@ void CRegisterView::OnPopupMenu(wxCommandEvent& event)
   CCodeWindow* code_window = cframe->g_pCodeWindow;
   CWatchWindow* watch_window = code_window->GetPanel<CWatchWindow>();
   CMemoryWindow* memory_window = code_window->GetPanel<CMemoryWindow>();
+  wxCommandEvent ev(wxEVT_JUMPTO_ADDRESS);
```

Comment:
You should define it in the ```case IDM_VIEWCODE``` if it's only used there. You'll need braces, though.